extra-files/split_logs.py

```diff
@@ -0,0 +1,101 @@
+# -*- encoding: utf-8 -*-
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+from xml.dom import minidom
+import datetime
+import argparse
+import os
+import re
+
+def get_logs(path):
+    logs = {} # filename -> lines[]
+    for file_name in os.listdir(path):
+        if not file_name.startswith("guest-test-"):
+            continue
+        with open(path + file_name, 'r') as f:
```

Comment:
```suggestion
        with open(os.path.join(path, file_name), 'r') as f:
```
or at least don't forget the `/` between...

Reply:
This is still an issue

Reply:
fixed